README.rst

```diff
@@ -1132,33 +1132,39 @@ achieves the move via a replication of the data objects found to the destination
 resource , followed by a trimming of each replica from the source.  We assume for our current
 purposed that all replicas are "good", ie have a status of "1" ::
 
+  import pprint, sys
   from irods.resource import iRODSResource
+  import irods.keywords as kw, irods.exception as ex, irods.test.helpers as h
```

Comment:
Is `irods.test.helpers` meant for use outside of test code?

Reply:
`make_session` belongs in irods.test for its great utility there.  I wouldn't mind making an alternate version of it, say `irods.lib.make_session`, where the arguably the parameter test_server_version could/should to` False`, This goes to #412

Reply:
> Is `irods.test.helpers` meant for use outside of test code?

`make_session` belongs primarily in irods.test for its great utility there.  I wouldn't mind making an alternate version of it, say `irods.lib.make_session`, where the arguably the parameter test_server_version could/should to` False`, This goes to #412

Reply:
@korydraughn Let me know if you think this looks better....

Reply:
That sounds like a good idea. If that function provides great utility, we should move it out of the test module and into a module that presents as being for general usage.

Given we already have an issue for that, let's handle that work in another PR.